includes/blocks/course-theme/class-prev-next-lesson.php

```diff
@@ -55,11 +55,16 @@ public function __construct( Prev_Lesson $prev, Next_Lesson $next ) {
 	/**
 	 * Renders the block.
 	 *
+	 * @param array  $attributes The attributes that were saved for this block.
+	 * @param string $content The content that is rendered by the inner blocks.
+	 *
 	 * @return string The block HTML.
 	 */
-	public function render() : string {
-		$prev = $this->prev->render();
```

Comment:
I think now we don't need the prev and next attributes in the class anymore, and we could remove them, right? 

Also, I still wonder if it makes sense to have this wrapper block. Maybe we could remove it for now, and leave it for a later moment when we finalize the header design?

I wonder it because maybe we can have an InnerBlocks containing it, and the other buttons that we have in the header, for example. Otherwise, we could create some unnecessary extra levels of blocks making it a little more complex to the users find where to add the buttons, and so on. WDYT?

Reply:
> I think now we don't need the prev and next attributes in the class anymore, and we could remove them, right?

Oh yes. I removed the prev and next from this class. 

> Also, I still wonder if it makes sense to have this wrapper block. Maybe we could remove it for now, and leave it for a later moment when we finalize the header design?

You're probably right. The only reason I see wrapper is useful to us is when we want to enforce some markup/design rules when user uses the prev and next blocks together. So, for now we definitely don't need it. I can remove the wrapper block if you prefer it that way. Let me know what you decide. 

> I wonder it because maybe we can have an InnerBlocks containing it, and the other buttons that we have in the header, for example. Otherwise, we could create some unnecessary extra levels of blocks making it a little more complex to the users find where to add the buttons, and so on. WDYT?

For users, lots of small blocks means more freedom, but also more complexity. Small number of grouped and prearranged blocks means less freedom, but less complexity. It's very hard to say which approach is better than the other. From product perspective I believe it's easier for the user to deal with small number of grouped blocks compared to lots of mini blocks that you need to arrange yourself.

Also the more granular control the user gets over the blocks the more room for mistakes they'll have while using those blocks.